tests/regression/56-floats/03-infinity_or_nan.c

```diff
@@ -0,0 +1,11 @@
+// PARAM: --enable ana.float.interval
+#include <stdio.h>
+
+void main()
+{
+    double data;
+    fscanf(stdin, "%lf", &data);
```

Comment:
i know, it was me coming up with that originally, however we are not checking for uninitialized-access. Therefore, we should probably leaf this out, and make minimal examples for our cases.

Reply:
Good point. Done.